### ISSUE TITLE
Getting Started

### ISSUE THREAD

**Issue by danekja** [mcgnly/bike-nav#1]
1. Get working dev enviroment for Android platform.
2. Create your first Android app and test its deployment on both simulator and real device.
3. Push to project to the repository as a base for future work.

**Comment by mcgnly** [comment]
OK, so I have android studio set up, figured out how to point it to the updated JDK and updated the SDK as well. When you say to create my first android app, do you mean in the sense of a hello-word like found [http://developer.android.com/training/basics/firstapp/creating-project.html](here)?

**Comment by danekja** [comment]
Yep, I believe its the best way get in touch with the technology. Shows some basic principles.
